pages/index.tsx

```diff
@@ -1,13 +1,11 @@
 import React, { useState } from 'react';
 import styles from '../styles/Home.module.css'
-import { NextPageContext  } from 'next';
-import Head from 'next/head';
-import Link from 'next/link';
+import { NextPageContext } from 'next';
 import Parser from 'rss-parser';
 import LazyLoad from 'react-lazyload';
-import Skeleton from 'react-loading-skeleton';
-import { Button, DivPx, Tag } from '@moai/core';
-import { HiOutlineExternalLink as externalLink } from 'react-icons/hi';
+import Layout from "../components/Layout";
+import { Entry, LoadingEntry } from '../components/Entry';
+import { Button } from '@moai/core';
 
 const FEEDS = [
```

Comment:
unuse variable

Reply:
It is used here:

https://github.com/webuild-community/federated-blog/pull/1/files#diff-aa98fd0757d0e1741503c50cfafb7726939d19819638bbe8e030a27adfec34a3R26

Reply:
ah my bad